src/schoology-extractor/schoology_extractor/api/paginated_result.py

```diff
@@ -0,0 +1,113 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+from __future__ import annotations
```

Comment:
This import is required in order to be able to annotate a return type of the class itself:
`def get_next_page(self) -> PaginatedResult`